app/_data/docs_nav_konnect.yml

```diff
@@ -15,6 +15,8 @@
           url: /private-connections/aws-privatelink/
     - text: Geographic Regions
       url: /geo/
+    - text: Regional Consumers
```

Comment:
We are going with the naming of "Consumers" as we are not introducing a new entity, but are expanding on the existing Consumer entity such that they can be defined in a central location per region and referenced across multiple control planes. Doc: https://docs.google.com/document/d/1k-d9FqAk4Jqo-e9-19IekqKQlNWfaFdgBNBuJGBw80k/edit?tab=t.0#heading=h.tn4g33ltt5z7